system-tests/tests/smoke/cre/cre_suite_test.go

```diff
@@ -35,6 +35,7 @@ func Test_CRE_Suite(t *testing.T) {
 		})
 
 		t.Run("[v2] HTTP trigger and action test", func(t *testing.T) {
+			t.Skip("Skipping flaky test")
```

Comment:
can you say more about it? have you asked @jinhoonbang about it?

Reply:
yes, Dmytro already let him know here https://chainlink-core.slack.com/archives/C07GQNPVBB5/p1757085817724369